sdk/src/main/scala/com/horizen/forge/ForgerList.scala

```diff
@@ -0,0 +1,36 @@
+package com.horizen.forge
+
+import scorex.core.serialization.{BytesSerializable, ScorexSerializer}
+import scorex.util.serialization.{Reader, Writer}
+
+case class ForgerList(forgerIndexes: Array[Int]) extends BytesSerializable {
+  override type M = ForgerList
+
+  def updateIndexes(indexToUpdate: Array[Int]): Unit = {
+    indexToUpdate.foreach(toUpdate => {
+      if (toUpdate < forgerIndexes.length) {
```

Comment:
what if `toUpdate` is out of bounds? Let's throw IndexOutOfBoundsException

Reply:
Done